setupbase.py

```diff
@@ -485,70 +464,3 @@ def _record_commit(self, base_dir):
                 ])
     return MyBuildPy
 
-
-
-#---------------------------------------------------------------------------
-# bdist related
-#---------------------------------------------------------------------------
-
-def get_bdist_wheel():
-    """Construct bdist_wheel command for building wheels
-    
-    Constructs py2-none-any tag, instead of py2.7-none-any
```

Comment:
Yup, that part became unnecessary a while ago. This function doesn't even do that part anymore.